netflix/src/components/common/PlayBtn.tsx

```diff
@@ -0,0 +1,23 @@
+import { PlayIcon } from '@/public/icon';
+
+interface PlayBtnProps {
+  width: number;
+  handlePlayBtn?: () => void;
+}
+
+export default function PlayBtn(props: PlayBtnProps) {
+  const { width, handlePlayBtn } = props;
+
+  return (
+    <button
+      style={{ width: `${width}rem` }}
+      className={`flex justify-center items-center h-[45px] gap-4 rounded-md bg-btn-gray`}
+      onClick={handlePlayBtn}>
```

Comment:
width 처럼 css 스타일을 외부에서 받아야 할 때 twMerge 라이브러리를 사용하시면 더 편리할 것 같아요!! 혹시 추가적인 css 스타일을 더 받아와야 할 때에도 더욱 유연하게 대처도 가능할 거 같아요 ㅎㅎ

```suggestion
   interface PlayBtnProps {
     handlePlayBtn?: () => void;
     className?: string;
}

export default function PlayBtn(props: PlayBtnProps) {
  const {  handlePlayBtn, className } = props;

  return (
    <button
      className={twMerge(`flex justify-center items-center h-[45px] gap-4 rounded-md bg-btn-gray`, className)}
      onClick={handlePlayBtn}>
```

외부에서 className prop 에 예를들어 w-[20rem] 이렇게만 적어주셔도 돼서 편리하고, 추가적인 스타일도 넘겨야 할 때 유연하게 대처할 수 있을 것 같습니다 ㅎㅎ